crates/rattler_pypi_interop/Cargo.toml

```diff
@@ -0,0 +1,77 @@
+[package]
+name = "rattler_pypi_interop"
+version = "0.1.0"
+categories.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+edition.workspace = true
+readme.workspace = true
+
+[features]
+default = ["native-tls"]
+native-tls = ['reqwest/native-tls']
+rustls-tls = ['reqwest/rustls-tls']
+
+[dependencies]
```

Comment:
Are all these dependencies actually used? It seems like a lot for just the types? You can check with `cargo machete`